src/lib/definitions.js

```diff
@@ -1215,6 +1219,8 @@ function getFHPaddingEntries(index)
         '∞':
         [
             define('Infinity[TO_LOCALE_STRING]()', LOCALE_INFINITY),
+            define('Infinity[TO_LOCALE_STRING]("ja")[SLICE_OR_SUBSTR]("-1")[0]', JAPANESE_INFINITY),
+            define('Infinity[TO_LOCALE_STRING]("ja").at("-1")', JAPANESE_INFINITY, AT),
```

Comment:
So, does `Infinity[TO_LOCALE_STRING]()` not work well in IE11 on Windows 7 in Japanese?

Reply:
I tested `Number.protorype.ToLocaleString()` in IE11 on Windows 7, and had [this issue](https://github.com/fasttime/JScrewIt/issues/44).

Reply:
The point is that the new definitions are more expensive than existing current one, so if the current definition works well there is no reason to add more.

Reply:
if you only consider IE 11 (which works differently on Windows 7 and Windows 10), that would be cheaper ("∞" is currently 4562 chars, and `Infinity["toLocaleString"]("ja")["slice"]("-1")[0]` is 4475 chars.) if possible, the `[0]` part can be removed for saving 3 chars further.

Reply:
I see, thanks. Yes, the `[0]` at the end of the definition can be removed.